bundles/org.openhab.binding.insteon/src/main/resources/device_features.xml

```diff
@@ -403,6 +403,26 @@
 	<command-handler command="OnOffType">NoOpCommandHandler</command-handler>
 	<poll-handler>NoPollHandler</poll-handler>
 </feature>
+<feature name="WirelessMotionSensorDawndusk">
```

Comment:
Can we use something more generic than "Dawndusk"? This assumes that the variable is associated with the sun rise/set.

Reply:
I can make it whatever you'd like, but I think this is the traditional way to refer to this feature on motion sensors going back to old X10 stuff. How about lightdark?

Reply:
changed to WirelessMotionSensorLightdark